plat/common/bootinfo_fdt.c

```diff
@@ -10,12 +10,12 @@
 #include <libfdt.h>
 
 #define ukplat_bootinfo_crash(s)		ukplat_crash()
+__u64 mem_base, mem_sz;
```

Comment:
This is a hack. You need to get this information from bootinfo in `libukrsi`. You can update its prototype to:
```
void uk_rsi_init(struct uk_bootinfo *bi)
```
We follow this pattern in other drivers too.